src/types/interfaces.ts

```diff
@@ -0,0 +1,68 @@
+import { IdGenerator } from "../misc/idGenerator"
+
+export type Position3D = [number, number, number]
+
+export type PeerData = {
+  id: string
+  position: Position3D
+  islandId?: string
+}
+
+export type Island = {
+  id: string
+  peers: PeerData[]
+  maxPeers: number
+  center: Position3D
+  radius: number
+  sequenceId: number
```

Comment:
I'm curious, what is sequenceId for?

Reply:
It is used to prioritize older islands when merging, if they are the same size